.github/workflows/ci.yml

```diff
@@ -80,6 +80,9 @@ jobs:
             - name: Install Symfony Flex
               run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
 
+            - name: Enable Symfony Flex
+              run: composer global config --no-plugins allow-plugins.symfony/flex true
+
```

Comment:
Why? What to does it bring?

Reply:
Please see previous build errors: https://github.com/jolicode/elastically/runs/7856878624?check_suite_focus=true#step:6:9

Reply:
Oh yes ! 
But IMHO it should goes to this repository directly.

Reply:
Nope. Flex is only installed in the CI.